Data_Structures_Answers.md

```diff
@@ -2,17 +2,31 @@ Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
 
+its just straight up number maniputlation and so its linear O(1)
```

Comment:
Sorry for not leaving a review, you did a great job here. Nothing to add from the previous week!